include/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+install ( DIRECTORY ../include/mls DESTINATION include
```

Comment:
Can we move this to the main `CMakeLists.txt`?  

I would prefer not to have stuff in `include` other than `include/mls`.  Headers that are only needed internally should go in `src`.